lib/CMakeLists.txt

```diff
@@ -33,7 +33,16 @@ endif()
 
 add_subdirectory(discord)
 
-include(libRocket.cmake)
+find_package(libRocket QUIET)
+if (${libRocket_FOUND})
+    message(STATUS "Using system installed version of libRocket")
+else()
+    message(STATUS "libRocket not found on system, using bundled")
+    IF(NOT IS_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/../cmake/external/rpavlik-cmake-modules/launcher-templates")
+        message(FATAL_ERROR "External submodules could not be found. Please make sure you have updated your submodules.")
+    endif()
+    include(libRocket.cmake)
```

Comment:
The way we usually do this stuff is so have the `find_package` stuff confined to the cmake file that defines the target so this should be moved to libRocket.cmake.

Reply:
Didn't do this yet because it would generate a lot of noise in the diff here because of the indentation change (unless I'm missing a clever way to move it), but if the last version now is okay I can just move it over before a final version.

Reply:
You don't need to indent everything if you just `return()` after your new code. Then the existing code does not need to be in an `else()` clause.